DB/src/test/java/io/deephaven/db/v2/QueryTableWhereTest.java

```diff
@@ -716,32 +713,25 @@ public void testWhereBoolean() {
         }
     }
 
-    private static class SleepCounter implements IntUnaryOperator {
-        final int sleepDurationNanos;
+    private static class InterruptingCounter implements IntUnaryOperator {
         long invokes = 0;
-        CountDownLatch latch = new CountDownLatch(1);
+        boolean interrupt;
 
-        private SleepCounter(int sleepDurationNanos) {
-            this.sleepDurationNanos = sleepDurationNanos;
+        private InterruptingCounter(boolean interrupt) {
+            this.interrupt = interrupt;
         }
 
         @Override
         public int applyAsInt(int value) {
-            if (sleepDurationNanos > 0) {
-                final long start = System.nanoTime();
-                final long end = start + sleepDurationNanos;
-                // noinspection StatementWithEmptyBody
-                while (System.nanoTime() < end);
-            }
-            if (++invokes == 1) {
-                latch.countDown();
+            if (++invokes == 1 && interrupt) {
```

Comment:
This might be better if we specified an interval to interrupt after, but what you've done is more than adequate for the current use case.